src/modules/todos/pages/TodosPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState } from "react";
+import { useSelector } from "react-redux";
+import { UILayout } from "../../../core/styles/shared/UILayout";
+import { AddNewItemForm } from "../components/AddNewItemForm";
+import { List } from "@mui/material";
+import { TodoItem } from "../components/TodoItem";
+
+// This page is the page on which all todo items, and form for adding and updating the item are displayed.
+// To get all todo items from the state hook called useSelector from react-redux is used
+// useState hook is used to store data of the selected item which is later on used for its updating
+interface ItemProps {
+  status: boolean;
+  task: string;
+  _id: string;
+}
+
+export const TodosPage: FC = () => {
+  const [editTask, setEditTask] = useState("");
+  const [editId, setEditId] = useState("");
+
+  const allTodos = useSelector((state: any) => state.allTodos?.allTodos);
+  const sortedItems = [...allTodos].reverse();
+
+  return (
+    <UILayout text="Today's To-Do List">
+      <AddNewItemForm
```

Comment:
It's a little confusing that this component is named `AddNewItemForm`, but it's also used for editing todos.